Dockerfile

```diff
@@ -69,6 +69,11 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 # Then, use a final image without uv
 FROM python:3.12-slim-bookworm
 
+# Install system dependencies, including libexpat1 and clean up the cache
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    libexpat1 \
```

Comment:
While we are at it can you please add the `libsndfile1` library?

- [ ] Also, can you please add this lines in the `back/Dockerfile` too?

This other `Dockerfile` is for dev development but also needs these dependencies.